app/services/manifest_builder.rb

```diff
@@ -4,8 +4,8 @@ class ManifestBuilder
 
   ##
   # @param [Resource] resource the Resource subject
-  def initialize(resource, auth_token = nil)
-    @resource = RootNode.for(resource, auth_token)
+  def initialize(resource, auth_token = nil, current_ability = nil)
```

Comment:
I can create a separate issue for this, but I suspect that it might be easier to offer keyword arguments for the `ManifestBuilder` constructor.

Reply:
:+1: I just didn't wanna change a bunch right now.